WooCommerce/Classes/ViewRelated/Dashboard/MyStore/ChartMarker.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 import Charts
-
+import UIKit
```

Comment:
This was required for the file to compile.

I guess Charts the Package doesn't (or can't?) implicitly import UIKit like Charts the Pod did.